dbmigrate/README.md

```diff
@@ -0,0 +1,15 @@
+# Purpose of module
+Providing the possibility to set up a database migrations using `Liquibase`
+
+#How
+Reacts to command line arguments (See `LiquibaseAutoBindModule` for a complete list)
+
+Scans the classpath for files named `migrations.xml` and then uses `Liquibase` to run all migrations.xml files in an undefined order.
```

Comment:
I think there is some special stuff with in our fatjar config that merges multiple migrations.xml-files. That might be worth mentioning. Without it I suspect that maven shade will just replace one file with another.